framework/src/main/java/dev/ikm/komet/framework/observable/ObservableEntity.java

```diff
@@ -224,7 +224,11 @@ private static class EntityChangeSubscriber implements Subscriber<Integer> {
         public void onNext(Integer nid) {
             // Do nothing with item, but request another...
             if (SINGLETONS.containsKey(nid)) {
-                get(nid);
+                if (Platform.isFxApplicationThread()) {
+                    get(nid);
+                } else {
+                    Platform.runLater(() -> get(nid));
+                }
```

Comment:
I believe you have a merge conflict with this file. 
I believe the main branch does the same thing as above if that make any sense.